src/App.js

```diff
@@ -1,24 +1,54 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Switch, Route, Redirect, Link } from 'react-router-dom';
 
-import { UserMain } from './containers/userMain/UserMain';
-import { Login } from './containers/login/Login';
 import { Appointments } from './containers/appointments/Appointments';
-import { Navbar } from './components/navbar/Navbar';
+import { Contacts } from './containers/contacts/Contacts';
 
 function App() {
+  const [appointments, setAppointments] = useState([]);
+  const [contacts, setContacts] = useState([]);
+
+  const addAppointment = (title, contact, date) => {
```

Comment:
`<App />` shouldn't be concerned with the implementation details of the pages. The hierarchy should look like this:

```
- <App>
   - <AppointmentsPage>
      - holds state and business logic
   - <ContactsPage>
      - holds state and business logic
```

What we'll need to do is:

1. Create a Page component for each page. This will be a "smart" component that has the state and business logic. Those will be passed to the presentational components.

Reply:
As per our conversation going to keep the contact and appointment arrays in App.js.